selfdrive/controls/radard.py

```diff
@@ -98,11 +103,12 @@ def get_RadarState(self, model_prob: float = 0.0):
       "vLead": float(self.vLead),
       "vLeadK": float(self.vLeadK),
       "aLeadK": float(self.aLeadK),
+      "aLeadTau": float(self.aLeadTau),
       "status": True,
       "fcw": self.is_potential_fcw(model_prob),
       "modelProb": model_prob,
       "radar": True,
-      "aLeadTau": float(self.aLeadTau)
+      "radarTrackId": self.identifier,
```

Comment:
do we need this?

Reply:
IMO it's useful for debugging - I've used that to track differences Tracks between two versions of radard

Reply:
can be a new PR then

Reply:
sure: https://github.com/commaai/openpilot/pull/29809